### DISCUSSION TITLE
What is "Use Compressed Talkgroups"

### DISCUSSION THREAD

**Discussion by grahampaull** [DSheirer/sdrtrunk#1685] [General]
Can anyone explain what "Use Compressed Talkgroups" is? I had a look through the Wiki but can't see it mentioned.

![image](https://github.com/DSheirer/sdrtrunk/assets/26274989/f99256b9-15ae-4591-aa08-293558870ded)

**Comment by DSheirer** [comment]
Some Hytera DMR systems employ compressed radio ID ranges to ensure no radio identifier is ever more than 7 decimal digits long (can be up to 8-digits with uncompressed).

See my notes here:
https://github.com/DSheirer/sdrtrunk/issues/1443
